### PR TITLE
Adds pan-tilt hat documentation

### PR DESCRIPTION
**Pull request by robmarkcole**
**Description:**
Documentation describes the use of the pan-tilt hat by pimoroni.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant/pull/9262) <9262>

### CODE REVIEW

File: source/_components/sensor.pan_tilt_phat.markdown [hunk 1]
```diff
@@ -0,0 +1,115 @@
+---
+layout: page
+title: "Pan-tilt HAT"
+description: "Instructions on add the Pimoroni Pan-Tilt hat to Home Assistant."
```
Comment (frenck) [inline source/_components/sensor.pan_tilt_phat.markdown]: **Spelling & Grammar**:
- `add` -> `adding`

File: source/_components/sensor.pan_tilt_phat.markdown [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+comments: false
+sharing: true
+footer: true
+logo: raspberry-pi.png
+ha_category: DIY
+ha_release: 0.53
+ha_iot_class: "Local Push"
+---
+
+The `pan_tilt_phat` platform allows you to control a [Pimoroni Pan-Tilt hat](https://shop.pimoroni.com/products/pan-tilt-hat) mounted on the raspberry-pi running Home Assistant. The platform [imports](https://pypi.python.org/pypi/pantilthat/0.0.4) the [pantilthat](https://github.com/pimoroni/pantilt-hat) package by Pimoroni. The hat consists of two servos which can pan and tilt a mounted camera with 180 degrees of motion (+/- 90 degrees) along each axis (vvertical and horizontal). This documentation assumes you have a [raspberry pi camera](https://home-assistant.io/components/camera.rpi_camera/) mounted on the hat, although other brands of camera could be used with some modification to the mounting. The platform adds a sensor which displays the current pan angle (there is currently a bug in the tilt angle)). The platform provides services for setting the angle of each servo, and a service to home the servos to zero degrees on both axis.
```
Comment (frenck) [inline source/_components/sensor.pan_tilt_phat.markdown]: **Spelling & Grammar**:
- `vvertical` -> `vertical`
- `although other brands of camera could be used` -> `although other camera brands could be used`
- Singular noun, should be plural: `axis`  -> `axes`

**Comment by robmarkcole** [comment]
Thanks @frenck changes made.! :-)

**Comment by frenck** [comment]
Thanks for fixing, @robmarkcole 🥇

**Comment by fabaff** [comment]
The Pull request in the main repository was closed. Please re-open this PR is your changes are ready to merge.

**Pull request closed without merging**